open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/etc/default.conf.template

```diff
@@ -3,31 +3,24 @@
 server {
 
     listen                443 ssl;
-    #listen                 80;
     server_name           ${NGINX_SERVER_NAME};
     ssl_certificate       /etc/nginx/ssl/tls.crt;
     ssl_certificate_key   /etc/nginx/ssl/tls.key;
     ssl_password_file     /etc/nginx/pass/pass.txt;
 
-
-    #root /var/www/;
-    #index index.html;
-
     # Force all paths to load either itself (js files) or go through index.html.
     location /api {
-        proxy_pass ${UI_API};
-        proxy_set_header Host $http_host;
-        proxy_ssl_verify       off;
+        proxy_pass              ${UI_API}/api;
+        proxy_set_header        Host $http_host;
+        proxy_ssl_verify        off;
         proxy_ssl_session_reuse on;
-        proxy_ssl_server_name on;
+        proxy_ssl_server_name   on;
     }
 
+    # Set the static UI as the default location to load
     location / {
-        proxy_pass ${UI_STATIC};
-        proxy_set_header Host $http_host;
-        proxy_ssl_verify       off;
-        proxy_ssl_session_reuse on;
-        proxy_ssl_server_name on;
+        proxy_pass              ${UI_STATIC};
```

Comment:
see overall review comment. this seems to work on openshift (insofar as I test). We need to understand what the difference is if it doesn't work for you?